.github/workflows/release-on-tag.yml

```
@@ -0,0 +1,96 @@
+      # Create GitHub Release with JReleaser
+      - name: Run JReleaser
+        uses: jreleaser/release-action@v2
+        env:
+          JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          JRELEASER_PROJECT_VERSION: ${{ steps.version.outputs.VERSION }}
+        with:
+          arguments: full-release
+
+      # Optional: Deploy to Nexus
+      - name: Deploy to Nexus
+        if: ${{ secrets.NEXUS_USER != '' }}
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)